assets/constants.ts

```diff
@@ -5337,6 +5337,23 @@ export default {
   "ANALYTICS_ID": "",
   "SITE_NAME_FOR_ANALYTICS": "",
 
+  // Data required for Firebase authentication.
+  //
+  // NOTE TO RELEASE COORDINATORS: Please change these to the production values,
+  // and change useEmulator to be false, before deploying to production.
+  "FIREBASE_ENVIRONMENT": {
```

Comment:
For release-swappable configuration, we prefer to store simple values (bool, str) so please use separate FIREBASE_EMULATOR_ENABLED, FIREBASE_ENV_API_KEY, etc. constants.